src/binaryutils.cpp

```diff
@@ -324,6 +337,27 @@ InferenceEngine::Blob::Ptr convertMatsToBlob(std::vector<cv::Mat>& images, const
     }
 }
 
+ov::runtime::Tensor convertMatsToBlob_2(std::vector<cv::Mat>& images, const std::shared_ptr<TensorInfo>& tensorInfo, bool isPipeline) {
+    switch (tensorInfo->getPrecision()) {
+    case InferenceEngine::Precision::FP32:
```

Comment:
is it new API?

Reply:
Left by mistake - will be taken care of when we remove old headers includes.